### ISSUE TITLE
Warranty and Protection plan - gray cards

### ISSUE THREAD

**Issue by Lakshmishri** [Netcentric/vg-macktrucks-com-rd#58]
- [Figma design](https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2220%3A90274&mode=dev)
- This can be implemented using existing [cards](https://www.macktrucks.com/trucks/lr-electric/) block with variation for `gray background`
- Design for mobile/tablet view is missing however the cards will be displayed after each other 
- Links will have the following[ states](https://www.figma.com/file/URJhT8pkntqc6h6v5wSLqx/Mack-Core-Design-Kit?type=design&node-id=307-22016&mode=design&t=H32De2eFj45PvBWz-4)

**Comment by shirin27** [comment]
Related to #64 , should be solved by that, only testing needed

**Comment by BeckyMedlin** [comment]
@DanielaPedrochevd Not sure why this one is flagged as "On Hold". Please confirm if the team is waiting on anything before they can action this ticket, or if it's only a dependency on #64, which is in the current sprint.

**Comment by DanielaPedrochevd** [comment]
We are only waiting for #64

**Comment by DanielaPedrochevd** [comment]
After discussing with the team, this issue has to be picked up, it's not only testing so we will pick it up in sprint 4 as planned. Already refined and estimated.

**Comment by DanielaPedrochevd** [comment]
Moved to sprint 5 and related to #120

**Comment by Andrei-Boiangiu** [comment]
confirmed as done
as tested on https://58-gray-cards--vg-macktrucks-com-rd--netcentric.hlx.page/drafts/syb/cards
